app/com/linkedin/drelephant/spark/fetchers/SparkRestClient.scala

```diff
@@ -135,6 +157,31 @@ class SparkRestClient(sparkConf: SparkConf) {
       }
     }
   }
+
+  private def getLogData(attemptTarget: WebTarget): Option[SparkLogDerivedData] = {
+    val target = attemptTarget.path("logs")
+    logger.info(s"calling REST API at ${target.getUri}")
```

Comment:
Maybe change this to something the following to make it more specific:

logger.info(s"calling REST API at ${target.getUri} to get eventlogs")